### PR TITLE
Manage team doc

### PR DESCRIPTION
**Pull request by arkigos**
New documentation for team management.
## Description

Adds a single new doc (within a new directory) that outlines all team-related management actions in the UI. 
## Motivation and Context

We were missing documentation for these actions. 
## How Has This Been Tested?

N/A
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/docs/getting-started-aws/index.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+***
+
+## Sign Up <a name="sign-up"></a>
+
+Like with all cloud providers, AWS requires you to sign up:
+
+1. Go to https://aws.amazon.com/free/.
+2. Click Sign In to the Console. This will start the process of signing up after you select "I am a new user."
+3. Go through adding your Contact Info, Payment Info, verifying your identity, choosing a support plan.
```
Comment (sibblegp) [inline docs/docs/getting-started-aws/index.md]: "and picking a support plan."

File: docs/docs/getting-started-aws/index.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+***
+
+## Sign Up <a name="sign-up"></a>
+
+Like with all cloud providers, AWS requires you to sign up:
+
+1. Go to https://aws.amazon.com/free/.
+2. Click Sign In to the Console. This will start the process of signing up after you select "I am a new user."
+3. Go through adding your Contact Info, Payment Info, verifying your identity, choosing a support plan. 
+4. Confirm your information and you should be good to go!
```
Comment (sibblegp) [inline docs/docs/getting-started-aws/index.md]: Mention the free tier.

File: docs/docs/getting-started-aws/index.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+[Learn more about pricing](https://aws.amazon.com/ec2/pricing/) for AWS.
+
+***
+
+## Subscribe to EC2 <a name="subscribe"></a>
```
Comment (sibblegp) [inline docs/docs/getting-started-aws/index.md]: As I said, I'm not sure this is true anymore but I may be wrong.

File: docs/docs/getting-started-aws/index.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+## Get your Access Key ID & Secret Access Key <a name="get-key"></a>
+
+In order for Koding to integrate with EC2, we'll need your Access Key ID and Secret Access Key. 
+
+> AWS provides a guide for getting your Access Key and ID: http://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html
+
+We've copied that guide here:
+
+1. Use your AWS account email address and password to sign in to the AWS Management Console.
+> If you previously signed in to the console with IAM user credentials, your browser might open your IAM user sign-in page. You can't use the user sign-in page to sign in with your root credentials. Instead, choose Sign in using AWS Account credentials near the bottom of the page to go to the account sign-in page. In the upper right of the console, choose the account name or number and then choose Security Credentials.
```
Comment (sibblegp) [inline docs/docs/getting-started-aws/index.md]: The final sentence is a step on its own and should be separate from this paragraph.

File: docs/docs/getting-started-aws/index.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+***
+
+## Create your Stack <a name="create-stack"></a>
+
+***[What is a Stack?](/docs/what-is-a-stack)***
+
+To create your Stack:
+
+1. [Login to Koding](https://koding.com/Teams/Select)
+2. Go to your [Dashboard](https://relepic.koding.com/Home) OR click [Stacks](https://relepic.koding.com/Home/Stacks) from your sidebar.
```
Comment (sibblegp) [inline docs/docs/getting-started-aws/index.md]: Links go to a specific team.  Also true below.

File: docs/docs/getting-started-aws/index.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+      tags:
+        Name: '${var.koding_user_username}-${var.koding_group_slug}'
+      user_data: |-
+        export DEBIAN_FRONTEND=noninteractive
+        apt-get update -y
+        apt-get -y install mysql-server
+```
+Make whatever changes you'd like to this script. Our [Stack Reference for AWS](https://www.terraform.io/docs/providers/aws/index.html) will help you see what is possible.
+
+> You won't need to worry about the aws_*_key variables. These will be automatically populated when you set up credentials in the next step.
```
Comment (sibblegp) [inline docs/docs/getting-started-aws/index.md]: aws_* not aws_*_

File: docs/docs/getting-started-aws/index.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+        apt-get update -y
+        apt-get -y install mysql-server
+```
+Make whatever changes you'd like to this script. Our [Stack Reference for AWS](https://www.terraform.io/docs/providers/aws/index.html) will help you see what is possible.
+
+> You won't need to worry about the aws_*_key variables. These will be automatically populated when you set up credentials in the next step. 
+
+***
+
+## Set your Credentials <a name="set-creds"></a>
```
Comment (sibblegp) [inline docs/docs/getting-started-aws/index.md]: I think there might be an error in the markdown here as this isn't properly highlighted.

File: docs/docs/getting-started-aws/index.md [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+
+To add your AWS credentials:
+
+1. From your Edit Stack screen (if you've just created the Stack, you will be on that screen by default) click on the Credentials tab
+2. On the bottom right of the Credentials screen, click Create New
+3. Name your Credentials whatever you like (pick something that will help you recognize them at a glance)
+4. Paste in the ID and Key that you obtained [earlier](#create-key)
```
Comment (sibblegp) [inline docs/docs/getting-started-aws/index.md]: Be more specific about Access Key and Secret Key, etc. and which step they obtained them from.  Reference the .csv file they downloaded.

File: docs/docs/getting-started-aws/index.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+5. Click Save
+6. You'll be taken back the Credentials tab where you'll see your new credentials listed by name. Click Use This & Continue to set them as the credentials for this stack. 
+
+***
+
+## Save your Stack and Initialize <a name="save-init"></a>
+
+Once you've created your Stack and set our credentials:
```
Comment (sibblegp) [inline docs/docs/getting-started-aws/index.md]: set your credentials

**Review by sibblegp — CHANGES_REQUESTED** [review]
Typos and other changes.

**Comment by gokmen** [comment]
ping.

**Pull request closed without merging**